regression/cbmc/Function_Pointer18/test.desc

```diff
@@ -1,11 +1,13 @@
 CORE
 main.c
 
-^EXIT=0$
-^SIGNAL=0$
-\[f2.assertion.1\] line [0-9]+ assertion 0: SUCCESS
-\[main.assertion.1\] line [0-9]+ assertion x == 1: SUCCESS
+^\*\*\*\* WARNING: no body for function main::fptr_call
+^\*\*\*\* WARNING: no body for function main::fptr_call\$0
+\[f2.assertion.1\] line [0-9]+ assertion 0: FAILURE
```

Comment:
Why is this now a failure?  If I have read the right test case, there is no way `f` can be set to `f2`, so the assert in `f2` cannot be reachable?